pypeit/core/wavecal/templates.py

```diff
@@ -534,6 +534,17 @@ def main(flg):
                        outroot, lowredux=False, ifiles=ifiles, chk=True,
                        normalize=True)
 
+    if flg & (2**21):  # WHT/ISIS
+        reid_path = os.path.join(resource_filename('pypeit', 'data'), 'arc_lines', 'reid_arxiv')
+        iroot = 'wht_isis_blue_1200_4800.json'
```

Comment:
I have been putting these JSON files in the Development suite repo, in part
because some of them are very large.  But if this is small, here is fine.